src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -110,9 +139,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'product_id' => $productId,
             ])
             ->add('manufacturer', ManufacturerType::class)
-            ->add('related_products', UnavailableType::class, [
+            ->add('related_products', EntitySearchInputType::class, [
                 'label' => $this->trans('Related products', 'Admin.Catalog.Feature'),
                 'label_tag_name' => 'h2',
+                'entry_type' => RelatedProductType::class,
+                'entry_options' => [
+                    'block_prefix' => 'related_product',
+                ],
+                'remote_url' => $this->router->generate('admin_products_v2_search_associations', [
+                    'languageCode' => $this->employeeIsoCode,
+                    'query' => '__QUERY__',
+                ]),
+                'min_length' => 3,
+                'filtered_identities' => $productId > 0 ? [$productId] : [],
+                'placeholder' => $this->trans('Search product', 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
                'placeholder' => $this->trans('Search product', 'Admin.Catalog.Help'),
```

Reply:
I'll fix the domain here, but the other modifications are not in the scope of this PR It's been stalling for a long time and I don't want to delay it for a wording that's not related This PR is about related products, not SEO tab, they just share a comment component which is why the `SEOType` file was updated a bit
We will do a full review of all the wording on the page in each tabs later anyway